python/cuml/dask/common/dask_df_utils.py

```diff
@@ -147,6 +147,7 @@ def to_pandas(df):
     meta = c.submit(get_meta, dfs[0])
     # Inserting this wait to stop race-condition in scheduler
     # Ref: https://github.com/dask/dask/issues/6027
-    wait(meta)
 
-    return dd.from_delayed(dfs, meta=meta.result())
+    meta_local = meta.result()
```

Comment:
I assume these new locals are here to ease debugging, not to make a functional change, right?

Reply:
That is correct. Specifically in this case, just making sure there's a local reference to `meta` being held in a variable so I can verify whether ad-hoc references might be causing an issue.